decouple.py

```diff
@@ -228,6 +228,7 @@ def __call__(self, value):
         transform = lambda s: self.cast(s.strip(self.strip))
 
         splitter = shlex(value, posix=True)
+        splitter.escape = ''  # Do not escape backslashes
```

Comment:
@henriquebastos I want to call your attention to this line -- it fixes the issue for me, but may introduce backwards incompatibilities. I'm not sure if you have other users relying on this behavior. I would be happy with just this change, but let me know if you want me to introduce it via a new optional command arg or other backwards compatible change.